src/ui/common/src/components/pages/workflow/id/index.tsx

```diff
@@ -324,14 +300,31 @@ const WorkflowPage: React.FC<WorkflowPageProps> = ({
       </Box>
 
       {currentNode.type !== NodeType.None && (
-        <AqueductSidebar
-          zIndex={10}
-          position={SidebarPosition.bottom}
-          getSideSheetTitle={getNodeLabel}
-          getSideSheetHeadingContent={getNodeActionButton}
-        >
-          {getDataSideSheetContent(user, currentNode)}
-        </AqueductSidebar>
+        <Drawer anchor="right" variant="persistent" open={true}>
```

Comment:
How come open is hardcoded to `true` here? Is it because we rely on the content of the sidesheet to be `null`/`undefined` when nothing is selected and therefore nothing gets rendered?

I'm curious just because it might be cleaner in the long run to have an explicit control rather than rely on state emptiness.

Reply:
The drawer's open/close state is actually controlled by whether or not a node is selected. to close, we simply reset the selectedNode.

So in our case, we can leave the node open since this code is wrapped in a conditional check to show the entire drawer.

We could also move this conditional check to be inside the open prop as well.